src/adapter/adapter.ts

```diff
@@ -73,5 +79,8 @@ export const adapter = new EmpathyAdapterBuilder()
   .onBeforeRequest(({ request }) => {
     delete request.scope;
   }, 'topRecommendations')
+  .onBeforeRequest(({ request }) => {
+    delete request.scope;
+  }, 'search')
```

Comment:
Do we truly want this? Aren't banners and promoted associated with a one or multiple scopes?

Reply:
the request is not returning banners or promoted if you send the scope

Reply:
As discussed with @tajespasarela, this parameter should not be removed, as banners and promoted should be defined per one or multiple scopes.